src/models/accounts.ts

```diff
@@ -24,6 +26,17 @@ export const BLANK_ACCOUNT: Account = {
   email: "",
 };
 
+/**
+ * @description model used to describe a biit account
+ */
+export interface PreviousUser {
+  fname: string;
+  lname: string;
+  email: string;
+  profileImage?: string;
+  commonMeetups: Meetup[];
+}
```

Comment:
Can you add a `BLANK_PREVIOUS_USER`  model here. I think that would help with displaying dummy data and testing.

Reply:
Yes that's a good call, I'll add that in then merge it